taxonomies/additives.txt

```diff
@@ -23245,12 +23744,46 @@ wikipedia:en: https://en.wikipedia.org/wiki/Steviol_glycoside
 
 en: E960a, Steviol glycosides from Stevia
 xx: E960a
+ar: E960a, جليكوسيدات ستيفيول من ستيفيا
+az: E960a, Steviyadan Steviol qlikozidləri
+bg: E960a, Стевиол гликозиди от Стевия
+ca: E960a, Glicòsids d'esteviol procedents d'estèvia
+cs: E960a, Steviol glykosidy ze Stévie
+da: E960a, Steviolglycosider fra Stevia
 de: E960a, Steviolglycoside aus Stevia
+el: E960a, Γλυκοσίδες στεβιόλης από Στέβια
 es: E960a, Glucósidos de esteviol procedentes de estevia
+et: E960a, Steviolglükosiidid Steviast
+fi: E960a, Stevioliglykosidit Steviasta
 fr: E960a, Glycosides de stéviol issus de Stevia
+he: E960a, גליקוזידים סטיביול מסטיביה
+hi: E960a, स्टीविया से स्टीविओल ग्लाइकोसाइड
+hr: E960a, Steviol glikozidi iz Stevije
```

Comment:
2025-03-07T21:50:07.0404208Z ERROR - duplicate_synonym - 25511 - hr:steviol-glikozidi-iz-stevije already is a synonym of hr:e960 for entry en:e960 (ingredients) - hr:steviol-glikozidi-iz-stevije cannot be mapped to entry en:e960a / hr:e960a